client/src/components/MissionConfig.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { Button, Select, MenuItem,  FormControl, Dialog, TextField, Stack, DialogContent, InputLabel, Typography } from "@mui/material";
+import { Add, Upload } from "@mui/icons-material"; 
+
+interface MissionConfigProps {
+    missionName: string;
+    location: [number, number];
+    date: Date;
+    launchAltitude: Number;
+    rocketProfile: string;
```

Comment:
Since this is a form I don't think it is necessary to have all of these props. In the future we will use this for editing old missions, for that we will only need an id to make a GET request with. So basically the parents click event will pass the `id` of the mission to this child for it to populate it with data.

Reply:
so for now maybe just replace these with `missionId?: string;`. Then you can get rid of the placeholders below for now, we'll add them back in the follow up ticket